app/src/main/java/it/niedermann/nextcloud/deck/persistence/sync/adapters/db/DeckDatabase.java

```diff
@@ -71,6 +73,7 @@
                 DeckComment.class,
                 Mention.class,
                 SingleCardWidgetModel.class,
+                StackWidgetModel.class,
         },
         exportSchema = false,
         version = 14
```

Comment:
Is the version already updated? Might lead to errors on upgrade otherwise. If you aren't sure, you can test it by uninstalling, deploying the current release and then just start your version. If it doesn't die, everything is fine.

Reply:
Database upgrade has been updated.